apps/web/pages/api/addresses/[address]/proof/index.ts

```diff
@@ -0,0 +1,30 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { getProofsByNamespaceAndAddress, ProofTableNamespace } from 'apps/web/src/utils/proofs';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (req.method !== 'GET') {
+    res.status(405).json({ error: 'method not allowed' });
+    return;
+  }
+  const { address, namespace } = req.query;
+  if (!address) {
+    return res.status(400).json({ error: 'address is required' });
+  }
+
+  try {
+    const content = await getProofsByNamespaceAndAddress(
+      address as string,
+      namespace as ProofTableNamespace,
+    );
+
+    if (content) {
+      console.log(content);
+    }
```

Comment:
```suggestion
```

Reply:
thanks actually this log is not needed, removing it all together :)